replayer/frame_diff.cpp

```diff
@@ -0,0 +1,328 @@
+/**
+ * Copyright (c) 2015-present, Facebook, Inc.
+ * All rights reserved.
+ *
+ * This source code is licensed under the BSD-style license found in the
+ * LICENSE file in the root directory of this source tree. An additional grant
+ * of patent rights can be found in the PATENTS file in the same directory.
+ */
+
+#include <algorithm>
+
+#include "frame.h"
+
+// TODO: Why is this necessary?
```

Comment:
What is this comment for? I vaguely recall it always being there but I'm confused so you can probably remove it.